js/src/carousel.js

```diff
@@ -278,7 +276,7 @@ class Carousel extends BaseComponent {
 
   _addTouchEventListeners() {
     const start = event => {
-      if (this._pointerEvent && PointerType[event.pointerType.toUpperCase()]) {
+      if (this._pointerEvent && (event.pointerType === POINTER_TYPE_PEN || event.pointerType === POINTER_TYPE_TOUCH)) {
```

Comment:
how about defining POINTER_TYPE_MOUSE and checking here simply that it's NOT the mouse?

but yes seems kosher even as it is

Reply:
I thought I'd go with the exact same checks, but we can change it later if needed :)